src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
I'm afraid there could be more cases that we unaware of, never used `originalEvent` though, so i can't judge here.
It is a documented property, plus having it in one event and not in the other, seems inconsistent.

Can we consider reverting problematic commit instead? Seem less breaky.